src/HW4/StudentTest.java

```diff
@@ -0,0 +1,40 @@
+package HW4;
+
+public class StudentTest {
+    public static void main(String[] args) {
+
+        Student student1 = new Student();
+        student1.mathAverageScore = 3;
+        student1.economicAverageScore = 4;
+        student1.languageAverageScore = 5;
+        student1.student_name = "Ivan";
+        student1.student_surname = "Ivanov";
+        student1.student_ID = 678354;
+
+        int num = 3;
+        int sum1 = (student1.mathAverageScore + student1.economicAverageScore + student1.languageAverageScore) / num;
```

Comment:
To avoid duplication it's a good idea to create method:
void avgMarkAllSubjects() {
    double result = (mathAvgMark + economicAvgMark + foreignLangAvgMark) / 3;
    System.out.println("The average mark for all 3 subjects for " + lastName + " is " + result);
  }